src/middlewared/middlewared/plugins/smb.py

```diff
@@ -389,14 +389,12 @@ async def configure(self, job, create_paths=True):
         LDAP server.
```

Comment:
Comment needs update:  The comment talks about skipping these steps if ldap passdb backend is used.